packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/RecordForm/formParts/CollectingLocality/index.js

```diff
@@ -82,11 +112,10 @@ class CollectingLocality extends PureComponent {
             type="text"
           />
         </Grid.Column>
-        {hasCoordinates ? (
-          <Grid.Column width={16}>
-            <Coordinates
-              latitude={latitude}
-              longitude={longitude}
+        {hasPositionData ? (
+          <Grid.Column width={11}>
+            <CollectingPosition
+              formValueSelector={formValueSelector}
```

Comment:
if you pass in the data (below) as props, you can remove `connect` from the other component. you'll still need to select the nested properties, but I think it's cleaner not to repeat the formValueSelection twice.

      coordinatesVerbatim,
      georeferenceSourcesText,
      position,
      verticalPosition,